dom.bs

```diff
@@ -4136,51 +4136,41 @@ when invoked, must run these steps:
 
  <li><p>Let <var>node1</var> be <var>other</var> and <var>node2</var> be <a>context object</a>.
 
- <li><p>Let <var>attr1</var> be null.
+ <li><p>Let <var>attr1</var> and <var>attr2</var> be null.
 
- <li>
-  <p>If <var>node1</var> is an <a>attribute</a>, then:
-
-  <ol>
-   <li><p>Set <var>attr1</var> to <var>node1</var>.
-
-   <li><p>If <var>attr1</var>'s <a for=Attr>element</a> is non-null, then set <var>node1</var> to
-   <var>attr1</var>'s <a for=Attr>element</a>.
-  </ol>
+ <li><p>If <var>node1</var> is an <a>attribute</a>, then set <var>attr1</var> to <var>node1</var>
+ and set <var>node1</var> to <var>attr1</var>'s <a for=Attr>element</a>.
 
  <li>
   <p>If <var>node2</var> is an <a>attribute</a>, then:
 
   <ol>
-   <li><p>Let <var>attr2</var> be <var>node2</var>.
+   <li><p>Set <var>attr2</var> be <var>node2</var> and <var>node2</var> to <var>attr2</var>'s
+   <a for=Attr>element</a>.
 
    <li>
-    <p>If <var>attr2</var>'s <a for=Attr>element</a> is <var>node1</var> and <var>attr1</var> is
-    non-null, then:
+    <p>If <var>node2</var> is <var>node1</var> and <var>attr1</var> is non-null, then:
 
     <ol>
      <li>
-      <p>For each <a>attribute</a> <var>node2Attribute</var> in <var>attr2</var>'s
-      <a for=Attr>element</a>'s <a for=Element>attribute list</a>:
+      <p>For each <a>attribute</a> <var>node2Attr</var> in <var>node2</var>'s
```

Comment:
Reading this anew, I also wonder if it'd be more clear to call *node2Attr* simply *attr* or something without a 2 in it, since node1==node2 here.